code/script.js

```diff
@@ -252,8 +281,12 @@ const setSecret = () => {
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
+  charactersInPlay = CHARACTERS;
   // What else should happen when we start the game?
+  setSecret();
+  generateBoard();
+  board.style.display = 'flex';
+  winOrLose.style.display = 'none';
```

Comment:
Decide if you want to use semicolons or not throughout your code, right now you alternate between using them or not. It would help solidify your code more.

Reply:
Please make sure you configure your VScode to do it for you 🤓